x-pack/packages/ml/aiops_components/src/document_count_chart/document_count_chart.tsx

```diff
@@ -200,7 +200,6 @@ export const DocumentCountChart: FC<DocumentCountChartProps> = (props) => {
 
   const { data, uiSettings, fieldFormats, charts } = dependencies;
 
-  const chartTheme = charts.theme.useChartsTheme();
```

Comment:
Something has changed with the document count chart in the ML AIOps pages (log rate analysis and log pattern analysis) where the chart type on table row selection / hover is now no longer stacked.

This PR:

<img width="1068" alt="image" src="https://github.com/elastic/kibana/assets/7405507/ebce6348-1057-43c5-9b85-a5ac7fb53988">


`main`:

<img width="1201" alt="image" src="https://github.com/elastic/kibana/assets/7405507/60dc6531-5145-46d6-a9db-d67a63cf7af4">

You can use the Kibana sample logs data set to test this one, but please reach out to @alvarezmelissa87 if you need help working through this regression.

Reply:
Ah thanks, this stacking on histogram charts was previously forced. Now after https://github.com/elastic/elastic-charts/pull/2225 we allow the chart to have clustered (ie. non-stacked) bars. Clustered bars are the default.

I updated this in f6dda7dc80378d4cc08eb23dd51c412d7d6da295, which now appears to render the correct charts. ⬇️ 

<img width="995" alt="image" src="https://github.com/elastic/kibana/assets/19007109/591d1487-3f58-4c8b-93d7-bdfcaa055cf1">

<img width="1004" alt="image" src="https://github.com/elastic/kibana/assets/19007109/712cd19d-2ed7-4f4b-a726-a48366fe7704">